app/services/redact_service.rb

```diff
@@ -1,23 +1,30 @@
 # Redact client_ids. Only needed as a sweep-up after LEP-193, so delete after
 # this has been run on both staging and production successfully (along with the test and rake task)
-class RedactClientIdService
+class RedactService
   class << self
     def call
       RequestLog.find_each do |li|
-        li.update!(request: filter_client_ids(li.request.deep_symbolize_keys))
+        li.update!(request: redact_data(li.request.deep_symbolize_keys))
       end
     end
 
   private
 
-    def filter_client_ids(hash)
+    def redact_data(hash)
+      submission_date = hash.dig(:assessment, :submission_date)
+      filter_client_ids(submission_date, hash)
+    end
+
+    def filter_client_ids(submission_date, hash)
```

Comment:
What do you think of using generic method name? e.g: filter_attributes or filter_payload